src/mococrw/openssl_wrap.h

```diff
@@ -1337,5 +1337,15 @@ enum class ellipticCurveNid
 };
 
 EC_KEY *_EVP_PKEY_get0_EC_KEY(EVP_PKEY *pkey);
+
+void _PKCS5_PBKDF2_HMAC(const char *pass, int passlen,
```

Comment:
The intend of this library layer is to provide as good memory management as possible in the situation with C++ and OpenSSL. This is, e.g., visible in all the places where it returns smart pointers to OpenSSL objects.

Consequently, I would prefer this function to already manage the output buffer and return a vector<uint_8>
I know that this pattern is not a 100% correct applied but no reason to make the overall situation worse.
Same for the other KDF

Reply:
done